examples/camera-app/linux/src/pushav-transport/pushav-transport.cpp

```diff
@@ -88,65 +88,61 @@ void PrintTransportSettings(PushAVClipRecorder::ClipInfoStruct clipInfo, PushAVC
 }
 
 void PushAVTransport::ConfigureRecorderSettings(const TransportOptionsStruct & transportOptions,
-                                                AudioStreamStruct mAudioStreamParams, VideoStreamStruct mVideoStreamParams)
+                                                AudioStreamStruct & audioStreamParams, VideoStreamStruct & videoStreamParams)
 {
-    bool debug = true; // Set this to false for debug purpose
+    bool debug = false; // Set this to true for debug purposes
 
     if (debug)
     {
-        clipInfo.mHasAudio    = true;
-        clipInfo.mHasVideo    = true;
-        clipInfo.mUrl         = transportOptions.url.data();
-        clipInfo.mTriggerType = static_cast<int>(transportOptions.triggerOptions.triggerType);
+        mClipInfo.mInitialDuration      = 20;
```

Comment:
Are these values to the various durations taken from any setting? At the very least would recommend defining them as constants instead of using magic numbers in the code.

Reply:
No, Those are hard-coded settings for debugging purpose, I just keep it as is. We should eventually remove this part and only keep the else portion.